package.json

```diff
@@ -49,6 +50,7 @@
     "@types/react": "^19.0.7",
     "@types/react-dom": "^19.0.3",
     "babel-loader": "^9.2.1",
+    "comlink": "^4.4.2",
```

Comment:
This should be added as a "dependency" rather than "devDependency" since Comlink is required at runtime

(and rerun `npm install`)